cdap-app-fabric/pom.xml

```diff
@@ -142,6 +142,18 @@
     </dependency>
     <dependency>
       <groupId>co.cask.cdap</groupId>
+      <artifactId>cdap-api-spark</artifactId>
```

Comment:
I see, I can try to move it to app-fabric-tests, or use a MapReduce instead. All I need is a SchedulableProgramType that is not WORKFLOW, to test that it returns a 400 bad request